css_styles/styles.css

```diff
@@ -0,0 +1,117 @@
+body {
+  margin: 0;
+  padding: 0;
+  background: url(https://cdn.pixabay.com/photo/2014/05/03/01/04/manhattan-336708_1280.jpg); 
+  background-size: cover;
+}
+
+.box{
+  width: 550px;
+  background: rgba(0, 0, 0, 0.5);
+  padding: 40px;
+  text-align: center; 
+  margin: auto;
+  margin-top: 5%;
+  color: white;
+  font-family: 'Century Gothic' , sans-serif; 
+}
+
+.box-img{
+  border-radius: 50%;
+  width: 280px;
+  height: 350px;
+}
+
+.box h1{
+  font-size: 50px;
+  letter-spacing: 4px;
+  font-weight: 100;
+  color: red;
+}
+
+.box h5{
+  font-size: 25px;
+  letter-spacing: 3px;
+  font-weight: 100;
+  padding: 15px;
+}
+
+.box p{
+  text-align: center;  
+  font-size: 17px; 
+  margin: 0px;
+ }
+
+ul{
+  margin: 1px;
+  padding: 0;
+}
+
+.box li{
+  display: inline-block;
+  margin: 8px;
+  list-style: none;
+  transition: all ease-in-out 250ms;
+}
+
+.box li a{
+  color: red;
+  text-decoration: none;
+  font-size: 70px;
+  list-style: none;
+}
+
+.box li a:hover{
+   color: #b9b9b9;
+
+}
+
+header {
+    text-align: center;
```

Comment:
Indent these lines and the closing `}` correctly

Same for the `h1` and `*` below

And actually looks like you don't have a `<header>` tag so these styles are unused and could be removed